lib/atat-web-api-stack.ts

```diff
@@ -59,14 +61,19 @@ export class AtatWebApiStack extends cdk.Stack {
 
   constructor(scope: Construct, id: string, props: AtatWebApiStackProps) {
     super(scope, id, props);
+
     const { environmentName, network } = props;
+    const sharedAtatKmsKey = new kms.Key(this, "AtatKey", {
+      enableKeyRotation: true,
+    });
```

Comment:
I know we mentioned rotating keys using lambda, is this the same or different from that?

Reply:
This is different -- I could try to explain it but honestly I think the graphics at https://docs.aws.amazon.com/kms/latest/developerguide/rotate-keys.html explain it a little better than I can. This is purely for the KMS Key, not for any sort of larger definition of "key"